src/comlink.ts

```diff
@@ -67,6 +67,8 @@ export type Remote<T> =
       ? Promise<boolean>
       : unknown
   ) & {
+    (...args: any): void;
+    new: (...args: any) => void;
```

Comment:
I don't think this is correct. Not every remote object will have a call and construct signature, and certainly not all taking `any` arguments and returning `void`. This should be dependent on the shape of `T`.

Reply:
Hello @felixfbecker, the remote objects _don't_ need to have a call signature or a construct signature. However, whether Comlink don't have these signatures, TypeScript will give you the following error when trying to construct a wrapper worker: "This expression is not constructable. ... (ts2351)". The construct signature is `...args: any` for matching any parameters and types. Finally, returning anything than `void` will give you the following error: "Only a void function can be called with the 'new' keyword. (ts2350)".